example/named_entity_recognition/README.md

```diff
@@ -15,4 +15,12 @@ To reproduce the preprocessed training data:
 
 To train the model:
 
-- `$ cd src && python ner.py`
\ No newline at end of file
+- `$ cd src && python ner.py`
+
+To run inference using trained model:
+
+1. Recreate the bucketing module using `sym_gen` defined in `ner.py`
+2. Loading saved parameters using `module.set_params()`
+
+Refer to the [Bucketing Module example](https://github.com/apache/incubator-mxnet/blob/master/example/rnn/bucketing/cudnn_rnn_bucketing.py#L167)
```

Comment:
Line numbers usually get obsolete and I have seen many broken links due to that. Can we just refer to cudnn_rnn_bucketing.py example and provide function name that users should read.

Reply:
Thanks for pointing out, addressed this